src/Camille.RiotGames/src/Util/RegionalRequester.cs

```diff
@@ -67,6 +95,19 @@ public RegionalRequester(IRiotGamesApiConfig config, string route)
             var retries = 0;
             var num429s = 0;
             var num5xxs = 0;
+
+            // If the region is not being used as a subdomain
+            if (_config.ApiCallRegionConfig == RegionConfig.InUrlAsRegionQueryParameter)
+            {
+                // Append the region as a query parameter
+                var uri = new UriBuilder(_client.BaseAddress.Scheme + "://" + _client.BaseAddress.Host + request.RequestUri);
+                var query = HttpUtility.ParseQueryString(uri.Query);
+                query[_config.RegionKey] = _region;
+                uri.Query = query.ToString();
+                // Replace the request with a new one that has the region correctly appended
+                request = new HttpRequestMessage(request.Method, uri.ToString());
```

Comment:
This would actually break any body params, (or header params if they exist, maybe with RSO, IDK)